components/bl-clock/README.md

```diff
@@ -11,8 +11,9 @@ This is a component for Backendless [UI-Builder](https://backendless.com/develop
 | Property                                     | Type                                                            | Default Value      | Logic                    | Data Binding | UI Setting | Description                                                                                                       |
 |----------------------------------------------|-----------------------------------------------------------------|--------------------|--------------------------|--------------|------------|-------------------------------------------------------------------------------------------------------------------|
 | Type: `type`                                 | Select [Clock: `clock`, Stopwatch: `stopwatch`, Timer: `timer`] | Clock: `clock`     |                          | NO           | YES        | Allows to determine the function of the component                                                                 |
-| Time Variant: `timeVariant`                  | Select [HH:MM:SS: `hhmmss`, HH:MM: `hhmm`, HH: `hh`]            | HH:MM:SS: `hhmmss` |                          | NO           | YES        | Allows to determine variant of displaying time                                                                    |
-| Stopwatch Scale: `stopwatchScale`            | Number                                                          | 3                  |                          | NO           | YES        | Allows to determine the scale of  stopwatch seconds                                                               |
+| Time Format: `timeFormat`                    | Select [HH:MM:SS: `hhmmss`, HH:MM: `hhmm`, HH: `hh`]            | HH:MM:SS: `hhmmss` |                          | NO           | YES        | Allows to determine the time format                                                                               |
+| Stopwatch Scale: `stopwatchScale`            | Number                                                          | 3                  |                          | NO           | YES        | Allows to determine the scale of stopwatch seconds                                                               |
+| Stopwatch Time Format: `stopwatchTimeFormat` | Select [SS: `ss`, MM:SS: `mmss`, HH:MM:SS: `hhmmss`]            | SS: `ss`           |                          | NO           | YES        | Allows to determine the time format of displaying stopwatch                                                       |
```

Comment:
I'm not sure. But for this moment it looks like we join two component that works in different ways but has one common feature - time.

At least if I want to use Clock - `Stopwatch Time Format` settings confused me

@snaqqqystas @StasivPavlo what do you think if we split it for two independent components?

Reply:
Which PRs should we merge?

Also, I do not clearly understand why we should merge them, as they will be overwritten by new code.

Reply:
i think, need to split in first on two separated components, to reduce misunderstandings

Reply:
@v-excelsior Because, for example, this PR is more about the Stopwatch than the Clock component. For me, splitting the components with their existing changes will be easier than doing them again later.

Now there are no more PRs so I'm splitting the components.

Reply:
@StasivPavlo you should take into account that this repository is public. The entire team and customers could see this code, and we should avoid showing not-ready code.

As for the point "easier to split existing changes" - they all still exist in your local branches.